src/vectorsynchronizer.js

```diff
@@ -96,7 +96,7 @@ olcs.VectorSynchronizer.prototype.createSingleCounterpart = function(olLayer) {
   var onRemoveFeature = goog.bind(function(feature) {
     var geometry = feature.getGeometry();
     var id = goog.getUid(feature);
-    if (goog.isDefAndNotNull(geometry) && geometry.getType() == 'Point') {
+    if (!geometry || geometry.getType() == 'Point') {
```

Comment:
What if the feature being removed previously had a geometry that was not a point? Isn't the `goog.asserts.assertInstanceof(bb, Cesium.Billboard)` assertion going to fail in that case?